plugins/bufferlines/navic.nix

```diff
@@ -45,12 +66,12 @@ in
     };
 
     lsp = {
-      autoAttach = helpers.defaultNullOpts.mkBool false ''
+      autoAttach = defaultNullOpts.mkBool false ''
```

Comment:
It seems option names haven't been converted to snake_case. [Checking upstream](https://github.com/SmiteshP/nvim-navic?tab=readme-ov-file#-customise), it seems they should be.

Reply:
i think I accidentally left it while testing the deprecations..